helpers.js

```diff
@@ -1,4 +1,16 @@
-const { access } = require("fs/promises");
+const { async: AsyncStreamZip } = require("node-stream-zip");
+const {
+  basename: resolveBaseName,
+  relative: resolveRelativePath,
+  resolve: resolvePath,
+} = require("path");
+const Archiver = require("archiver");
+const fs = require("fs");
+
+const { lstat, access, mkdir } = fs.promises;
```

Comment:
Is this approach documented anywhere, or did it just happen to work?

Reply:
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api